app/models/user.rb

```diff
@@ -16,6 +16,7 @@ class User < ApplicationRecord
   has_many   :miq_widget_contents, :dependent => :destroy
   has_many   :miq_widget_sets, :as => :owner, :dependent => :destroy
   has_many   :miq_reports, :dependent => :nullify
+  has_many   :service_orders, :dependent => :nullify
```

Comment:
I agree that when we delete a user we want to keep the service_orders around for things like reporting on order history per tenant, so `nullify` makes sense here.  However, we will lose any reference to the user in that object.  Does it make sense to add a column like "user_name", similar to the "requester_name" used in miq_requests?  (Note: Since we do have a link between the service_model and miq_requests we could also find the user that way.)

@dclarizio @jrafanie thoughts?